### ISSUE TITLE
Add a testimonial page for the training class

### ISSUE THREAD

**Issue by majesticjam** [ifpusa/ifp-public#15]
Cody Stephens
cody.stephens@arconic.com
Testimonial 
This is a great course for people who are new to or experienced with hydraulics. I learned a lot during this course but the major key for me was the troubleshooting side. As a mechanical technician, most of my work has been with everything but hydraulics and this class really helped me feel more comfortable with my troubleshooting capabilities. 

It has been a week since this class that I am typing this and I have already used some of the mental tools I was given during our visit to IFP. I strongly agree that this is a great course for anyone especially if you are not working on hydraulic systems frequently. A quick refresher (8 hr. class) could save a lot of downtime or unnecessary costs with hiring an expert for something "simple." 

Jack was very knowledgeable in what he taught and what types of things can really bite you if you aren't paying attention. I hope to take the next class after this one or maybe even take this class again a few years down the road! Thank you for the time and effort put into the materials we covered and thank you for allowing my company in the door!

**Comment by susodapop** [comment]
That's great! I need to write up some markup for how we will present the testimonials page. I have a few ideas for how we could include testimonials for the classes. 

Examples: Netlify has a great testimonial slider on their marketing page [here](https://www.netlify.com/).

![screen shot 2018-12-12 at 1 53 59 pm](https://user-images.githubusercontent.com/17067911/49895343-cd7ba180-fe15-11e8-8c6c-862311bbbc49.png)


 I've included a picture but you really should try it yourself. It slides from left to right when viewed on a touch device.

Another example I found of interest was on this [personal blog](https://www.stavros.io/) from a commenter on HackerNews. Notice how he uses a speech-bubble shaped callout for each of his blog posts. The callout gives a quick summary of the content in the post. We could do something like this for the training class. But it might also be a good fit for our careers page -- we could have a current employee in that role / department give us a one-liner that we can position near the link for the job posting (see pictures).

![screen shot 2018-12-12 at 1 56 45 pm](https://user-images.githubusercontent.com/17067911/49895355-d7050980-fe15-11e8-8b58-82af14a454e9.png)

**Comment by majesticjam** [comment]
Regarding the Netlify format with headshot icon, we do not have an image for the Stephens testimonial nor any of the previous attendees. Should I reach back out to him and see if he is willing to send us a selfie we could edit? Also, should we perhaps take head shots of our attendees for the next class on Tuesday? What about a specific disclosure of usage? is something like that required?

**Comment by susodapop** [comment]
**Reaching back out**: I wouldn't do that at this time since we don't have a place to put the image on the site. If we ask for such a picture, he'll expect to see it on the site which is a promise we can't keep right now.

**Headshots of class attenders**: I can see this getting really awkward really quickly. The most could imagine would be taking a group picture of the class at the end or some *incidental* photography while the session is taught. I figure the typical student of our class isn't likely to appreciate out-of-the-blue photography.

**Disclosure**: We definitely need something signed saying that we can use someone's picture on the site.

Overall
=====
I think we'd be better served to just use first name and last initial to identify our class testimonials. Anything beyond that could become a headache quickly.

**Comment by majesticjam** [comment]
I would agree, external people head shots would be high maintenance, awkward and a potential violation of privacy. I like the simple first name/last initial assignation as well. In the classes I attended offsite, I asked the coordinator if we could take pictures and use them for marketing promotion purposes. There was no written agreement.
